neutrino.go

```diff
@@ -1360,15 +1356,16 @@ func (s *ChainService) outboundPeerConnected(c *connmgr.ConnReq, conn net.Conn)
 	// If we're already connected to this peer, then we'll close out the new
 	// connection and keep the old.
 	if s.PeerByAddr(peerAddr) != nil {
-		conn.Close()
+		s.connManager.Remove(c.ID())
```

Comment:
We may want to eventually change the names of `Remove/Disconnect` in the `connmgr` to better reflect their retry semantics.